src/action.js

```diff
@@ -38,9 +50,24 @@ export async function run() {
 
   const rwClient = client.readWrite
 
+  let tweetOpts = {}
+
+  try {
+    const media_ids = await uploadMedia(rwClient, media, mediaAltText)
```

Comment:
let's move the condition which runs this code or not here, not inside the uploadMedia function